configure

```diff
@@ -3017,12 +3015,6 @@ else
 fi
 
 
-# Check whether --enable-vmthreads was given.
-if test "${enable_vmthreads+set}" = set; then :
-  enableval=$enable_vmthreads;
-fi
-
```

Comment:
I'm obviously in favour of merging this, but should we make `--enable-vmthreads` be a configure-time error if the option is passed? That'll help package maintainers clean up their configure flags if there's an actual error at build time for a release.